static/app/views/starfish/modules/databaseModule/queries.ts

```diff
@@ -380,15 +380,20 @@ export const useQueryMainTable = (options: {
   });
 };
 
-export const useQueryTransactionByTPM = (row: DataRow) => {
+export const useQueryTransactionByTPMAndP75 = (
+  transactionNames: string[]
+): DefinedUseQueryResult<
+  {count: number; interval: string; p75: number; transaction: string}[]
+> => {
   const pageFilter = usePageFilters();
   const {startTime, endTime} = getDateFilters(pageFilter);
   const dateFilters = getDateQueryFilter(startTime, endTime);
-  const queryFilter = `group_id = '${row.group_id}'`;
+  const queryFilter = `transaction IN ('${transactionNames.join(`', '`)}')`;
 
   const query = `
   select
```

Comment:
big nit lol but I see we're switching the case of operations in these queries, i.e `SELECT` vs `select` and we should stick to one or the other

Reply:
Yeah 👍, its a fair point, we're all over the places with these queries, i'll update them all at once in another PR.